config/development.json

```diff
@@ -94,6 +94,7 @@
 		"layout/query-selected-editor": true,
 		"legal-updates-banner": true,
 		"login/magic-login": true,
+		"login/qr-code-login": true,
```

Comment:
I might also enable it in horizon, just to have a way to check it once deployed :) (via horizon.wordpress.com)

Reply:
I think enabling it on wpcalyso first and then when we are close to launching we can enable it on horizon. 
This way we can test it out on the calypso live links.